app/main/views/suppliers.py

```diff
@@ -444,3 +443,28 @@ def create_your_account_complete():
         "suppliers/create_your_account_complete.html",
         email_address=email_address
     ), 200
+
+
+@main.route('/<framework_slug>/opportunities-overview', methods=['GET'])
```

Comment:
This seems a weird place and URL for this page.

Might be better in `briefs.py` where all of the other `/opportunities` routes are and live under something like `/opportunities/overview`?

Or even in `frameworks.py` where all the '/frameworks/<framework_slug>' routes are and live under something like `'/frameworks/<framework_slug>/opportunities-overview'?

And I know Roz has gone now, but channelling her spirit, I'm not sure "overview" is a good word to have in a URL at all, so just `/frameworks/<framework_slug>/opportunities`. Would be fine here (because this is the supplier app the full URL would end up being `/suppliers/frameworks/<framework_slug>/opportunities`, which makes sense - i.e. the opportunities of interest to X supplier for Y framework).